codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/s3/S3ErrorWith200StatusIntegration.kt

```diff
@@ -40,15 +40,9 @@ class S3ErrorWith200StatusIntegration : SwiftIntegration {
         // Instead of playing whack-a-mole broadly apply this interceptor to everything but streaming responses
         // which adds a small amount of overhead to response processing.
         val output = ctx.model.expectShape(operationShape.output.get())
-        val outputIsNotAStreamingBlobShape =
-            output.members().none {
-                val targetShape = ctx.model.expectShape(it.target)
-                val isBlob = it.isBlobShape || targetShape.isBlobShape
-                val isStreaming = it.isStreaming || targetShape.isStreaming
-                isBlob && isStreaming
-            }
+        val outputIsNotAStreamingShape = output.members().none { ctx.model.expectShape(it.target).isStreaming }
```

Comment:
In the logic above:
- We no longer check if the targeted shape is a blob, since we want to keep the customization off of both data _and_ event streams.
- No need to check for the trait on the member shape, since `@streaming` is never allowed to be applied there, only on the target shape.